src/analyses/ai.cpp

```diff
@@ -82,6 +82,158 @@ void ai_baset::output(
 
 /*******************************************************************\
 
+Function: ai_baset::output_json
+
+  Inputs: The namespace and goto_functions
+
+ Outputs: The JSON object
+
+ Purpose: Output the domains for the whole program as JSON
```

Comment:
I think there's not much overlap. @thk123's code is concerned with converting the goto program to json while here we're converting abstract states.